Staking/contracts/Stacking.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract StakingContract {
+    address public owner;
+    uint public lockUpPeriod; // Lock-up period in seconds
+    uint public interestRate; // Annual interest rate
```

Comment:
How to set interest rate? Can it not change after initialisation?

Reply:
can I set 0.09 as interest rate here?

Reply:
I mean it can change, and the owner has the rights to do that, I'll add a function for this...for the second question do you mean 0.09%?

Reply:
yes

Reply:
No, we'd need a fixed-point number for that, or else we could define a practice in the contract and set a base unit for it like 0.001 ==  1, then we could add 90 for 0.09, and so on, and at the place of division we could divide it by 1000.